src/main/frontend/extensions/pdf/assets.cljs

```diff
@@ -29,12 +34,13 @@
 
               :else
               (util/node-path.join
-               "file://"                                  ;; TODO: bfs
-               (config/get-repo-dir (state/get-current-repo))
-               "assets" filename))]
+                "file://"                                   ;; TODO: bfs
+                (config/get-repo-dir (state/get-current-repo))
+                "assets" filename))
+        ext-name (util/get-file-ext filename)]
```

Comment:
get-file-ext needs to return a lower-cased result, otherwise `.PDF` will not work.

Reply:
`util/get-file-ext` function has handled it.

Reply:
My bad 😂